config/non-serializable/mailer.js

```diff
@@ -3,4 +3,8 @@ module.exports = {
     host: `${process.env.MAILER_HOST || 'localhost'}`,
     port: `${process.env.MAILER_PORT || 25}`,
   },
+  auth: {
+    user: `${process.env.MAILER_AUTH_USER || 'user'}`,
```

Comment:
I think the default here should be empty (`''` or `null` or `undefined`?) as our `localhost` servers don't have authentication being unreachable from the outside world.

Reply:
ok - sounds sensible as it could result in an unreadable error message

Reply:
@giorgiosironi From the docs... https://nodemailer.com/smtp/#authentication

> If authentication data is not present, the connection is considered authenticated from the start. Otherwise you would need to provide the authentication options object.

So that makes me wonder how `''`or `null` would be interpreted / sent to the server :thinking:

Reply:
https://github.com/nodemailer/nodemailer/blob/33b62e2ea6bc9215c99a9bb4bfba94e2fb27ebd0/lib/smtp-transport/index.js#L88-L97 seems to say that the default should be `{}` for the whole object? Needs to be tested.